fhir-persistence/src/test/resources/logging.unitTest.properties

```diff
@@ -49,4 +49,4 @@ java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
 ############################################################
 
 # Set this to FINE or higher to output the SQL statements and related debug info
-com.ibm.fhir.persistence.level = INFO
+com.ibm.fhir.persistence.level = FINE
```

Comment:
did you mean to commit this change?

Reply:
No, and once you are done reviewing, I'll revert back.  Thanks for the catch.

Reply:
Fixed in next commit